reference/checkouts.v3.yml

```diff
@@ -400,9 +401,8 @@ paths:
                         name: Chambray Towel
```

Comment:
Please verify `parent_id` type. I think it should be an integer or null.

Reply:
changed it to null